src/posthog-surveys.ts

```diff
@@ -302,7 +193,13 @@ export class PostHogSurveys {
             return this.instance.featureFlags.isFeatureEnabled(value)
         })
     }
-    getNextSurveyStep = getNextSurveyStep
+    getNextSurveyStep(survey: Survey, currentQuestionIndex: number, response: string | string[] | number | null) {
+        if (isNullish(assignableWindow.__PosthogExtensions__?.getNextSurveyStep)) {
+            logger.warn('init was not called')
+            return 0
+        }
```

Comment:
Lets do better logging here similar to https://github.com/PostHog/posthog-js/pull/1663
so we know if the problem is the __PosthogExtensions__ not loaded, or the getNextSurveyStep somehow wasn't available